app/views/projects/index.html.erb

```diff
@@ -65,6 +65,7 @@ See docs/COPYRIGHT.rdoc for more details.
           <tr>
             <%= render partial: 'hierarchy_header' %>
             <%= projects_sort_header_tag('name', caption: Project.human_attribute_name(:name), param: :json) %>
+            <%= projects_sort_header_tag('status', caption: Project.human_attribute_name(:status), param: :json) %>
```

Comment:
This results in an error when trying to sort by status.
<img width="1202" alt="Bildschirmfoto 2019-10-17 um 09 17 32" src="https://user-images.githubusercontent.com/7457313/66986323-fff1cd00-f0be-11e9-8956-addbf8b91b24.png">

Reply:
Thanks. Added the order capability